pkg/datapath/linux/config/config_test.go

```diff
@@ -48,6 +50,22 @@ var (
 	ipv6DummyAddr = netip.MustParseAddr("2001:db08:0bad:cafe:600d:bee2:0bad:cafe")
 )
 
+func testParams() loader.Params {
```

Comment:
Could we do a single `TestLoader` helper in `loader/loader_test.go` that assembles the args and calls `newLoader()`?

Reply:
I'll think about it. We'll likely change this many times over in follow-up PRs that add the loader reconciliation loop, but I'll try to make this as clean as I can already.

Reply:
Add `NewLoaderForTest` to `loader/loader.go` and using it from both `loader_test.go` and `config_test.go` (it can't go into `loader_test.go` as then it's not accessible from `config_test.go`).